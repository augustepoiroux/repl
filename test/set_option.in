{"cmd": "import Lean\nopen Lean Elab Command\n\nelab \"#show_pp\" : command => do\n  let o ← getOptions\n  logInfo m!\"Elab.async = {Elab.async.get o}\"\n\n#show_pp"}

{"cmd": "import Lean\nopen Lean Elab Command\n\nset_option Elab.async true\n\nelab \"#show_pp\" : command => do\n  let o ← getOptions\n  logInfo m!\"Elab.async = {Elab.async.get o}\"\n\n#show_pp"}

{"cmd": "import Lean\nopen Lean Elab Command\n\nelab \"#show_pp\" : command => do\n  let o ← getOptions\n  logInfo m!\"Elab.async = {Elab.async.get o}\"\n\n#show_pp", "setOptions":[[["Elab","async"],true]]}

