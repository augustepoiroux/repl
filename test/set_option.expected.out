{"messages":
 [{"severity": "info",
   "pos": {"line": 8, "column": 0},
   "endPos": {"line": 8, "column": 8},
   "data": "Elab.async = false"}],
 "env": 0}

{"messages":
 [{"severity": "info",
   "pos": {"line": 10, "column": 0},
   "endPos": {"line": 10, "column": 8},
   "data": "Elab.async = true"}],
 "env": 1}

{"messages":
 [{"severity": "info",
   "pos": {"line": 8, "column": 0},
   "endPos": {"line": 8, "column": 8},
   "data": "Elab.async = true"}],
 "env": 2}

